pop/src/commands/PopAccountDisableCommand.cpp

```diff
@@ -193,7 +193,18 @@ void PopAccountDisableCommand::GetActivities()
 {
 	CommandTraceFunction();
 
-	m_client.SendRequest(m_getActivitiesSlot, "com.palm.activitymanager", "list", MojObject());
+	MojObject payload(MojObject::TypeObject);
```

Comment:
Here you can drop MojObject::TypeObject
The rest looks good !

Reply:
Updated